Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml

```diff
@@ -0,0 +1,525 @@
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 260,
+          "y": 1390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: 00aac347-43a2-4b79-8677-798d7dcd5794
+    type: regular
+    task:
+      id: 00aac347-43a2-4b79-8677-798d7dcd5794
+      version: -1
+      name: Set ASN type - Cloud ASN
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "36"
+    scriptarguments:
+      key:
+        simple: ASNType
+      value:
+        simple: CloudASN
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 260,
+          "y": 1210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1195,
+        "width": 1300,
+        "x": -160,
+        "y": 260
+      }
+    }
+  }
+inputs:
+- key: ResolveIP
+  value:
+    simple: "True"
+  required: false
+  description: Determines whether to convert the IP address to a hostname using a DNS query (True/ False).
+  playbookInputQuery:
+- key: InternalRange
+  value: {}
+  required: false
+  description: "A list of internal IP ranges to check IP addresses against. \nFor IP Enrichment - Generic v2 playbook."
+  playbookInputQuery:
+outputs:
+- contextPath: IP
+  description: The IP objects
+  type: unknown
+- contextPath: DBotScore
+  description: Indicator, Score, Type, Vendor
+  type: unknown
+- contextPath: Account
+  description: The account object.
+  type: unknown
+- contextPath: IAM
+  description: Generic IAM output.
+  type: unknown
+- contextPath: UserManagerEmail
+  description: The email of the user's manager.
+- contextPath: UserManagerDisplayName
+  description: The display name of the user's manager.
+  type: unknown
+- contextPath: MSGraphUser
+  description: The MSGraphUser object.
+  type: unknown
+- contextPath: MSGraphUserManager.Manager
+  description: The MSGrMSGraphUserManageraph Manager object.
+  type: unknown
+- contextPath: SailPointIdentityNow
+  description: The SailPointIdentityNow object.
+  type: unknown
+- contextPath: SailPointIdentityNow.Account
+  description: The IdentityNow account object.
+  type: unknown
+- contextPath: IdentityIQ
+  description: The IdentityIQ object.
+  type: unknown
+- contextPath: ActiveDirectory.Users
+  description: The ActiveDirectory Users object.
+  type: unknown
+tests:
+- No tests (auto formatted)
+marketplaces: ["marketplacev2"]
```

Comment:
The pack already only for XSIAM, you can remove it from here

Reply:
done